src/interfaces/ILBPairExt.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+import {ILBFlashLoanCallback} from "./ILBFlashLoanCallback.sol";
+
+interface ILBPairExt {
+    function swap(bool swapForY, address to) external returns (bytes32 amountsOut);
+
+    function flashLoan(ILBFlashLoanCallback receiver, bytes32 amounts, bytes calldata data) external;
+
+    function mint(address to, bytes32[] calldata liquidityConfigs, address refundTo)
+        external
+        returns (bytes32 amountsReceived, bytes32 amountsLeft, uint256[] memory liquidityMinted);
+
+    function burn(address from, address to, uint256[] calldata ids, uint256[] calldata amountsToBurn)
+        external
+        returns (bytes32[] memory amounts);
```

Comment:
`mint` and `burn` are actively used by users that manage liquidity actively which is a common case with concentrated liquidity.   
consider moving them back to the LBPair contract.   
another way to reduce contract size is to set `optimizer_runs` in foundry.tomlto a lower value, e.g. 600.